account_bank_receipt/report/report.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+  Copyright (C) 2014-2019 Akretion (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  The licence is in the file __manifest__.py
+-->
+
+<odoo>
+
+    <report id="report_account_bank_receipt"
+            model="account.bank.receipt"
+            string="Check Deposit"
```

Comment:
```suggestion
            string="Bank Receipt"
```
**Note** I think wording check deposit should change to "bank receipt" on all places.